casper/src/main/scala/coop/rchain/casper/util/rholang/RuntimeManager.scala

```diff
@@ -268,29 +261,35 @@ class RuntimeManager[F[_]: Concurrent: ToAbstractContext] private (
 
   private def injAttempt(
       deploy: Deploy,
-      reducer: ChargingReducer[Task],
-      errorLog: ErrorLog[Task]
+      reducer: ChargingReducer[F],
+      errorLog: ErrorLog[F]
   ): F[(PCost, Vector[Throwable])] = {
     implicit val rand: Blake2b512Random = Blake2b512Random(
       DeployData.toByteArray(ProtoUtil.stripDeployData(deploy.raw.get))
     )
-    ToAbstractContext[F].fromTask(
-      for {
-        result    <- reducer.inj(deploy.term.get).attempt
-        phlos     <- reducer.phlo
-        oldErrors <- errorLog.readAndClearErrorVector()
-        newErrors = result.swap.toSeq.toVector
-        allErrors = oldErrors |+| newErrors
-      } yield (CostAccount.toProto(phlos) -> allErrors)
-    )
+    for {
+      result    <- reducer.inj(deploy.term.get).attempt
+      phlos     <- reducer.phlo
+      oldErrors <- errorLog.readAndClearErrorVector()
+      newErrors = result.swap.toSeq.toVector
+      allErrors = oldErrors |+| newErrors
+    } yield (CostAccount.toProto(phlos) -> allErrors)
   }
 }
 
 object RuntimeManager {
   type StateHash = ByteString
+
+  import monix.eval.Task
+  import monix.execution.Scheduler
   import coop.rchain.catscontrib.TaskContrib._
 
   def fromRuntime(active: Runtime[Task])(implicit scheduler: Scheduler): RuntimeManager[Task] =
+    fromRuntime[Task](active).unsafeRunSync
```

Comment:
do we need the non-monadic `fromRuntime` anymore?

Reply:
maybe `Task[RuntimeManager[Task]]`?

Reply:
Yes, it's still used in tests